src/common/Gateway.sol

```diff
@@ -483,26 +307,5 @@ contract Gateway is Auth, Recoverable, IGateway {
     function _outboundBatchSlot(uint16 centrifugeId, PoolId poolId) internal pure returns (bytes32) {
```

Comment:
Just for oganization purposes, could we add
```

    //----------------------------------------------------------------------------------------------
    // Helpers
    //----------------------------------------------------------------------------------------------
```

below `endBatching()`?

And potentially, also change `Outgoing` to `Outgoing handlers`, and add
```

    //----------------------------------------------------------------------------------------------
    // Outgoing payments
    //----------------------------------------------------------------------------------------------
```

above `subsidizePool`

Reply:
Agree with the helpers. Regarding the outgoing payments, not sure given it's everything very related. For example `repay()`/`addUpnpaindMessage()` will go into "outgoing handler" or "outgoing payments"? 🤔

Reply:
Yeah fair